Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -448,8 +448,8 @@ extension _FileManagerImpl {
     func isExecutableFile(atPath path: String) -> Bool {
 #if os(Windows)
         return (try? path.withNTPathRepresentation {
-            var dwBinaryType: DWORD = 0
-            return GetBinaryTypeW($0, &dwBinaryType)
+            // Use SHGetFileInfo instead of GetBinaryType because the latter returns the wrong answer for x86 binaries running under emulation on ARM systems.
+            return (SHGetFileInfoW($0, 0, nil, 0, numericCast(SHGFI_EXETYPE)) & 0xFFFF) != 0
```

Comment:
What do you think of adding `SHGFI_EXETYPE` to `WinSDK+Extensions.swift` to avoid the `numericCast`?

Reply:
Done